test/unit/entwine.js

```diff
@@ -1,17 +1,45 @@
 import assert from 'assert';
-import { HttpsProxyAgent } from 'https-proxy-agent';
 import View from 'Core/View';
 import GlobeView from 'Core/Prefab/GlobeView';
 import Coordinates from 'Core/Geographic/Coordinates';
 import EntwinePointTileSource from 'Source/EntwinePointTileSource';
 import EntwinePointTileLayer from 'Layer/EntwinePointTileLayer';
 import EntwinePointTileNode from 'Core/EntwinePointTileNode';
+import sinon from 'sinon';
+import Fetcher from 'Provider/Fetcher';
 import Renderer from './bootstrap';
 
+import ept from '../data/entwine/ept.json';
+import eptHierarchy from '../data/entwine/ept-hierarchy/0-0-0-0.json';
+
+const urlEpt = 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/pointclouds/entwine/ept.json';
+const urlEptHierarchy = 'https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/pointclouds/entwine/ept-hierarchy/0-0-0-0.json';
```

Comment:
Maybe declare another const for `https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/pointclouds/entwine/`

Reply:
Done